src/Components/Notes/NoteCard.jsx

```diff
@@ -158,16 +159,11 @@ export default function NoteCard({
    * @param {string} accessToken
    * @param {number} commentId
    */
-  // TODO(pablo)
-  /* async function deleteComment(commentId) {
-    // TODO(pablo): handle response
-    await deleteCommentGitHub(repository, commentId, accessToken)
-    const newComments = comments.map((comment) => ({
-      ...comment,
-      synched: (comment.id !== commentId) && comment.synched,
-    }))
+  async function deleteCommentGithub(commentId) {
+    await deleteComment(repository, commentId, accessToken)
```

Comment:
I think we planned to use return statuses to set comments instead of filtering, like in https://github.com/bldrs-ai/Share/issues/1200

Reply:
@pablo-mayrgundter getting 204 with the data of the response object undefined.

ChatGPT:
A 204 No Content status code indicates that the server successfully processed the request, but is not returning any content. This is typical for a POST request where the server does not return a response body.